examples/twilio_api_v2010.yaml

```diff
@@ -239,6 +239,12 @@ paths:
           description: Created
       security:
         - accountSid_authToken: [ ]
+      x-twilio:
+        conditional:
+        - - TestInteger
```

Comment:
Why does this have double dashes?

Reply:
This is needed since conditional attribute is a list of list, and we need to pick any one of the properties from each list.
so for example we can have the properties like this 
`{
  "x-twilio": {
    "conditional": [
      [
        "url",
        "twiml",
        "application_sid"
      ],
	[
        "
        “Foo”
	“bar”
      ]
    ]
  }
}`

representing this would require double dashes in yaml